conf/evaluator.env

```diff
@@ -3,3 +3,5 @@ POSTGRESQL_PASSWORD=ve_db_user_evaluator_pwd
 EVALUATOR_TOPIC=vulnerability.evaluator.upload,vulnerability.evaluator.recalc
 VMAAS_VULNERABILITIES_API=/api/v1/vulnerabilities
 PROMETHEUS_PORT=8085
+WORKER_THREADS=30
```

Comment:
this is adding variables with duplicate name as are already in listener.env, there is a test checking duplicate variables, let's move them to common.env or if there is a need to have them called differently in evaluator and listener, please rename

Reply:
seems good now